src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -36,6 +36,20 @@
           + "  AND draft != true ")
   List<DataAccessRequest> findAllDataAccessRequests();
 
+
+  /**
+   * Find all non-draft DataAccessRequests for the given datasetId including only the dar data needed for the metrics page
+   *
+   * @return List<DataAccessRequest>
+   */
+  @SqlQuery(
+      "SELECT id, reference_id, draft, user_id, create_date, sort_date, submission_date, update_date, "
+          + "(data #>> '{}')::jsonb AS data FROM data_access_request "
+          + " WHERE not (data #>> '{}')::jsonb ??| array['partial_dar_code', 'partialDarCode'] "
+          +  "AND draft = false" 
```

Comment:
I think we can move to querying only on draft status here and in the other non-draft query. We started with the `partial_dar_code` check because we didn't have draft at the time. Draft was added, and we never removed the other check.